build.psm1

```diff
@@ -1971,8 +1971,10 @@ function Start-DevPowerShell {
             Remove-Item env:DEVPATH
         }
 
-        if ($ZapDisable) {
-            Remove-Item env:COMPLUS_ZapDisable
+        if (Get-Variable -Name ZapDisable -ValueOnly -ErrorAction SilentlyContinue) {
+            if ($ZapDisable) {
+                Remove-Item env:COMPLUS_ZapDisable
+            }
```

Comment:
The extra `if` block shouldn't be needed, I don't think?

If the variable exists, `Get-Variable -ValueOnly` will return its value as-is, whether that's true or false (or anything else, for that matter).

If the variable doesn't exist, it'll return `$null` which is treated by `if` statements the same way as `$false`. 🙂

Reply:
Thank you @vexx32. I've made the change and pushed it.